prow/cmd/deck/static/common/rerun.ts

```diff
@@ -0,0 +1,151 @@
+          <i class='rerunModal-openOut material-icons state triggered' style='color: gray'>open_in_new</i>
+          </a>)
+        </div>
+        <div class="rerunModal-radioButtonRow">
+          <label class="rerunModal-radioLabel mdl-radio mdl-js-radio mdl-js-ripple-effect" for="rerunOriginalOption">
+            <input type="radio" id="rerunOriginalOption" class="mdl-radio__button" name="rerunOptions">
+            <span class="mdl-radio__label">Original Configuration</span>
+          </label>
+          (<a href="${originalURL}" target="_blank">View YAML
+          <i class='rerunModal-openOut material-icons state triggered' style='color: gray'>open_in_new</i>
+          </a>)
+        </div>
+      </div>
+      <div class="rerunModal-accordion">
+        <button class="rerunModal-accordionButton">
+          <i class='rerunModal-expandIcon material-icons state triggered' style='color: gray'>expand_more</i>
+          kubectl command
+        </button>
+        <div class="rerunModal-accordionPanel">
+          <div class="accordion-panel-content">
+            <p class="rerunModal-commandDescription">${getCommandDescription(LATEST)}</p>
+            <div class="rerunModal-commandContent">
+              <div class="rerunModal-command">${getCommand(commandURL)}</div>
+              <a class="rerunModal-copyButton mdl-button mdl-js-button mdl-button--icon">
+                <i class='material-icons state triggered' style='color: gray'>content_copy</i>
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+    `;
+
+    const rerunLatestOption = rerunElement.querySelector('#rerunLatestOption');
+    const rerunCommand = rerunElement.querySelector('.rerunModal-command');
+    const rerunCommandDescription = rerunElement.querySelector('.rerunModal-commandDescription');
+    rerunLatestOption.addEventListener('click', () => {
+      commandURL = latestURL;
+      rerunCommandDescription.innerHTML = getCommandDescription(LATEST);
+      rerunCommand.innerHTML = getCommand(commandURL);
+    });
+    const rerunOriginalOption = rerunElement.querySelector('#rerunOriginalOption');
+    rerunOriginalOption.addEventListener('click', () => {
+      commandURL = originalURL;
+      rerunCommandDescription.innerHTML = getCommandDescription(ORIGINAL);
+      rerunCommand.innerHTML = getCommand(commandURL);
+    });
+    const rerunCopyButton = rerunElement.querySelector('.rerunModal-copyButton');
+    rerunCopyButton.addEventListener('click', () => {
+      copyToClipboardWithToast(getCommand(commandURL));
+    })
+    const accordion = rerunElement.querySelector('.rerunModal-accordionButton');
+    const accordionPanel = rerunElement.querySelector('.rerunModal-accordionPanel');
+    const expandIcon = rerunElement.querySelector('.rerunModal-expandIcon');
+    accordion.addEventListener('click', () => {
+      if (!accordionPanel.classList.contains('rerunModal-accordionPanel--expanded')) {
```

Comment:
I think I'd be OK with a dual-class-name query like `.rerunModal-accordionPanel.expanded` if you are.

Then, you'd only be toggling the `.expanded` classname.

Note: I've seen CSS class-name conventions say things like "CSS class-names manipulated by Javascript should prefixed with `js-`, so it'd end up being `.js-expanded` in code and `.rerunModal-accordionPanel.js-expanded` in CSS.